capstone/backend/src/main/java/com/google/spanner/LibraryFunctions.java

```diff
@@ -124,6 +124,21 @@ public static void createUser(long userId, String userName, String email)
     dbClient.write(Arrays.asList(mutation));
   }
 
+  public static List<String> getItemTypes() throws SpannerException {
```

Comment:
where are you using this function?

Reply:
Using this function in ListPage.js. Looping through the GET requests on the front-end turns out to be a lot more of a hassle.

Reply:
If this goes through, I can just use axios.get('/api/v1/get-item-types') in ListPage.js

Reply:
Why is it a hassle? You can just change this line here to use your new function right? 
https://github.com/googleinterns/step39-2020/blob/cf006d133b9f3b9c6d89a2b2f21a71377c08f084/capstone/backend/src/main/java/com/google/servlets/GetItemTypesServlet.java#L53

Reply:
(and just ignore the page number param, since you're not using that).

Reply:
You can change the page param to be Optional: https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html

Reply:
Ended up handling it in the servlet! I might come back and add this if I have time later, but I don't think it's necessary right now if I changed the API to its original function